internal/gapicgen/cmd/genbot/github.go

```diff
@@ -76,11 +77,13 @@ type PullRequest struct {
 	IsOpen  bool
 	Number  int
 	Repo    string
+	IsDraft bool
 }
 
 // GithubClient is a convenience wrapper around a github.Client.
 type GithubClient struct {
-	c *github.Client
+	c   *github.Client
+	cql *githubv4.Client
```

Comment:
Consider naming this `cv4` instead of `cql`. There can in theory be GitHub API v5 in the future, which may or may not use GraphQL. See https://github.com/shurcooL/githubv4/issues/35 for relevant background.

Reply:
Done.